tests/2.Encoding.ts

```diff
@@ -357,7 +359,12 @@ describe('encoding', () => {
     it('should parse null', () => {
       const input = 'null';
 
-      for (const intDecoding of ['default', 'safe', 'mixed', 'bigint']) {
+      for (const intDecoding of [
+        algosdk.IntDecoding.DEFAULT,
+        algosdk.IntDecoding.SAFE,
+        algosdk.IntDecoding.MIXED,
+        algosdk.IntDecoding.BIGINT,
```

Comment:
v3 will actually keep the `algosdk.IntDecoding` enum as an argument to [`parseJSON`](https://github.com/algorand/js-algorand-sdk/blob/482cfe15553821dfcd357246e6b787b02f89fd2d/src/utils/utils.ts#L30). The change will be that we no longer expose this argument for configuration through the REST requests -- requests will always choose `BIGINT`, then the typed models will either keep the fields as bigints or downgrade to numbers depending on the type.

This changed in this PR because when I converted the file to typescript, it was unhappy dealing with an array of strings (even though the enum members are just strings).

Reply:
Sounds good, I guess you're keeping the argument so v2 can keep working as is.